### PR TITLE
feat: feature toggles for thorchain swapper

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Adds feature toggle for the following:
* thorchain swapper streaming swaps (off currently)
* thorchain swapper affiliate fees set to 25 bps with hidden checkbox (off currently)

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

* Low risk of broken thorchain swapper
* Low risk of broken donation checkbox logic for other swappers

## Testing

See above

### Engineering

☝️

### Operations

☝️

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5131** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5131" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5131?utm_source=stack-comment).

File: src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts [hunk 1]
```diff
@@ -28,8 +29,9 @@ export const thorchainApi: Swapper2Api = {
   ): Promise<Result<TradeQuote2[], SwapErrorRight>> => {
     const { receiveAddress } = input
 
-    // TEMP: thorchain swapper is set at 25 bps
-    const affiliateBps = '25'
+    const thorSwapAffiliateFees = getConfig().REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES
```
Comment (0xApotheosis) [inline src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts:32]: Maybe a boolean-like name,

```suggestion
    const applyThorSwapAffiliateFees = getConfig().REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES
```

File: src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx [hunk 2]
```diff
@@ -97,6 +98,7 @@ export const TradeInput = memo(() => {
   const buyAsset = useAppSelector(selectBuyAsset)
   const sellAsset = useAppSelector(selectSellAsset)
   const { isModeratePriceImpact, priceImpactPercentage } = usePriceImpact()
+  const thorSwapAffiliateFees = useFeatureFlag('ThorSwapAffiliateFees')
```
Comment (0xApotheosis) [inline src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx:101]:
```suggestion
  const applyThorSwapAffiliateFees = useFeatureFlag('ThorSwapAffiliateFees')
```

**Review by 0xApotheosis — APPROVED** [review]
Non-blocking comments, otherwise looks good.

**Comment by 0xApotheosis** [comment]
Nice, get 'er in.

**Pull request merged**